news/3694.bugfix

```diff
@@ -0,0 +1 @@
+Replace ReverseCheckBoxFieldWidget with CheckBoxFieldWidget in SearchControlPanelForm
```

Comment:
@danalvrz release notes should focus on what the end result is for the user, not on how the implementation was done. Also please include your github username for credit. So I would suggest:
```suggestion
Change the search control panel to select types not searched instead of searchable types. This fixes an inconsistency with Volto. [danalvrz]
```